GitHubMainApplication/sortOperation.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace GitHubMainApplication
+{
+    public class SortOperation
+    {
+        public SortOperation(int[] array) {
+            if (array.Length != 0)
+            {
+                Console.WriteLine("1.Ascending order\n2.Descending order");
+                int choice = Convert.ToInt32(Console.ReadLine());
+                switch (choice)
+                {
+                    case 1:
+                        SortAscending(array);
+                        break;
+                    case 2:
+                        SortDescending(array);
+                        break;
+                }
+            }
+            else
+            {
+                Console.WriteLine("array is empty");
+            }
+
+        }
+        int temperoryVariable,count=1;
+
+        public  void SortAscending(int[] array) {
+            for (int i = 0; i < array.Length; i++)
```

Comment:
apply appropriate naming conventions

Reply:
have appropriate naming convention except i and j in for loop